apps/earn-protocol-landing-page/components/layout/LandingMasterPage/LandingMasterPage.tsx

```
@@ -1,20 +1,34 @@
+import Image from 'next/image'
+import { usePathname } from 'next/navigation'
 
 import { NavigationWrapper } from '@/components/layout/Navigation/NavigationWrapper'
 import { useLandingPageData } from '@/contexts/LandingPageContext'
 
 import landingMasterPageStyles from '@/components/layout/LandingMasterPage/landingMasterPage.module.css'
 
+import palmLeft from '@/public/img/beach-club/palm_1.png'
+import palmRight from '@/public/img/beach-club/palm_2.png'
+
 interface LandingMasterPageProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)